### DISCUSSION TITLE
How to compile Mac binaries from a Windows 10 machine?

### DISCUSSION THREAD

**Discussion by BladePoint** [BowlerHatLLC/vscode-as3mxml#643] [Q&A] [answered]
Is it possible to compile Mac binaries from a Windows 10 machine? I have successfully created an "Adobe AIR: package Windows release (captive runtime)" from the "Run Build Task..." menu, and I would like to create a Mac version of the same thing. Do I need to go out and get a Mac or set up a Mac Virtual Machine to do that?

**Comment by joshtynjala** [comment, marked as answer]
> Do I need to go out and get a Mac or set up a Mac Virtual Machine to do that?

Yes. You must compile macOS binaries on a macOS machine. Similarly, macOS cannot compile Windows binaries, and requires a Windows machine.
